tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/WwwAuthenticateParametersIntegrationTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.NetFx.HeadlessTests
+{
+    [TestClass]
+    public class WwwAuthenticateParametersIntegrationTests
```

Comment:
>WwwAuthenticateParametersIntegrationTests [](http://example.com/codeflow?start=17&length=41)

good to see these e2e tests, wondring how stable some of those values are from graph and keyvault. time will tell I suppose.

Reply:
we'll find out.